### PR TITLE
Updating uninstall instructions for Oct release

### PR DESCRIPTION
**Pull request by jt000**
**Note:** Please wait for 👍 to merge _(release not available on download page yet...)_

### CODE REVIEW

File: docs/reporting-services/report-server-sharepoint/deploy-report-viewer-web-part.md [hunk 1]
```diff
@@ -104,31 +96,66 @@ Get-SPWebApplication "<web application url>" | Get-SPSite -Limit ALL |
 
 Although SharePoint Central Administration provides solution retraction, you do not need to retract the **ReportViewerWebPart.wsp** file unless you are systematically troubleshooting an installation or patch deployment problem.
 
+### Disable the **Report Viewwer Web Part** feature in each SharePoint site with the Report Viewer Web Part enabled
```
Comment (guyinacube) [inline docs/reporting-services/report-server-sharepoint/deploy-report-viewer-web-part.md]: Typo - Viewer

File: docs/reporting-services/report-server-sharepoint/deploy-report-viewer-web-part.md [hunk 2]
```diff
@@ -104,31 +96,66 @@ Get-SPWebApplication "<web application url>" | Get-SPSite -Limit ALL |
...
+
+    ![Site settings from the gear icon.](media/sharepoint-site-settings.png)
+
+    By default, SharePoint web applications are accessed through port 80. This means that you can often access a SharePoint site by entering *http://<computer name>* to open the root site collection.
+
+2. Under **Site Collection Administration**, select **Site collection features**.
+
+3. Click **Deactivate** next to the **Report Viewer Web Part** feature
+
+### Retract the solution from the Farm
```
Comment (guyinacube) [inline docs/reporting-services/report-server-sharepoint/deploy-report-viewer-web-part.md]: Farm should be lower case.

File: docs/reporting-services/report-server-sharepoint/deploy-report-viewer-web-part.md [hunk 3]
```diff
@@ -104,31 +96,66 @@ Get-SPWebApplication "<web application url>" | Get-SPSite -Limit ALL |
...
+
 1. In SharePoint Central Administration, in **System Settings**, select **Manage farm solutions**.
 
 2. Select **ReportViewerWebPart.wsp**.
 
 3. Select Retract Solution.
 
-### Remove the web part from Site settings
+### Removing the Solution in PowerShell
```
Comment (guyinacube) [inline docs/reporting-services/report-server-sharepoint/deploy-report-viewer-web-part.md]: Solution should be lowercase

**Comment by jt000** [comment]
Closing pull request. Unfortunately, not going to be able to introduce this functionality into this release due to issues this introduced with the Install-SPSolution cmdlet.

**Pull request closed without merging**